base/show.jl

```diff
@@ -1132,7 +1132,8 @@ function show_nd(io::IO, a::AbstractArray, limit, print_matrix, label_slices)
     end
     tail = size(a)[3:end]
     nd = ndims(a)-2
-    function print_slice(idxs...)
```

Comment:
You'll want to change the `return`s in this function to `continue` since it's now a for loop.